cfme/common/provider_views.py

```diff
@@ -576,3 +576,14 @@ def before_fill(self, values):
         for widget in self.COND_WIDGETS:
             if values.get(widget):
                 getattr(self, widget).fill(values.get(widget))
+
+
+class TemplatesCompareView(InfraProvidersView):
+    """Compare Templates page."""
+    title = Text('.//div[@id="center_div" or @id="main-content"]//h1')
```

Comment:
TODO: add `BaseNonInteractiveEntitiesView` as a nested view here, so its method for checking entities against a context list is available.

Reply:
As we determined in chat, TemplatesCompareView does not use BaseNonInteractiveEntitiesView . I have added comparison_table to TemplatesCompareView.